std/math.d

```diff
@@ -5549,7 +5549,7 @@ real NaN(ulong payload) @trusted pure nothrow @nogc
     }
 }
 
-@safe pure nothrow @nogc unittest
+pure nothrow @nogc unittest // not @safe because taking address of local.
```

Comment:
You will need to annotate the test as `@system` - since a couple of months all Phobos unittest need to be either `@system` or `@safe`, s.t. (1) it can't be forgotten to add `@safe` (and avoid regressions) and (2) s.t. it's easy to grep for all unsafe stuff (and make the according tests `@safe`).